config/settings.json

```diff
@@ -13,6 +13,9 @@
           "0.22": "21da7d7e2b5e2680cd9d2e0a2fb4a7d13d8baa9d"
         }
       }
+    },
+    "Setonix": {
+
```

Comment:
👀 

It's ok that this is empty?

Reply:
WHOOPS that was for testing.

Reply:
Removed as part of https://github.com/ACCESS-NRI/build-cd/pull/137/commits/7daa1eccdfb69e26528129e5a41f6d414a13fe07